pyhdb/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 apilevel = "2.0"
 threadsafety = 2
-paramstyle = "format"
+paramstyle = ("qmark", "named", "format")
```

Comment:
I don't know if this paramstyle definition is conform with PEP 249.
We should discuss this with db-sig@python.org